ansible/roles_ocp_workloads/ocp4_workload_kubevirt/tasks/workload.yml

```diff
@@ -13,6 +13,10 @@
     state: present
     definition: "{{ lookup('file', 'operatorgroup.yaml') }}"
 
+- name: Waiting 5 seconds to ensure the operator group is created
```

Comment:
We should not wait an arbitrary amount of time like this. If you want to see if the operator group (or anything) is created, add a task that retrieves the `operatorgroup` and runs retries until it is there.

Reply:
I agree with you. I've changed it.